src/components/TrustedBy.tsx

```diff
@@ -39,6 +39,14 @@ const StyledTrustedBy = styled.section`
             grid-template-columns: repeat(1, 1fr);
         }
     }
+
+    .fcc {
+        transform: scaleY(1.3);
```

Comment:
Please always preserve aspect ratios for logos (i.e. never just `scaleX()` or `scaleY()`). If you need it bigger or smaller, please use `scale()`.

```suggestion
        transform: scale(1.3);
```

(Or remove this line if you didn't want the logo wider.)